dist/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
@GutsKTA what's the goal of this tag?

Reply:
some of google font have to load data not only from "https://fonts.googleapis.com", also from other servers, as I understand. But I see this link at first time and I didn't even notice when I was coping link from google fonts.
![gstatic](https://user-images.githubusercontent.com/67595127/103158199-0bad4900-47c4-11eb-8764-b1b6da8afad5.PNG)